.gitattributes

```diff
@@ -0,0 +1 @@
+*.whl filter=lfs diff=lfs merge=lfs -text
```

Comment:
needed for [this](https://github.com/grafana/oncall/pull/4495/files#diff-ed9dcc8e8a654115a66ebd30f4c8eedc42ea9426a23f9e305149f03b6c964f44), otherwise when trying to commit the new `.whl` file (which was slightly over 100MB), I ended up with this:
```bash
❯ git push origin jorlando/update-uwsgi-version
Uploading LFS objects: 100% (1/1), 113 MB | 0 B/s, done.
Enumerating objects: 13, done.
Counting objects: 100% (13/13), done.
Delta compression using up to 10 threads
Compressing objects: 100% (9/9), done.
Writing objects: 100% (10/10), 107.23 MiB | 17.12 MiB/s, done.
Total 10 (delta 5), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (5/5), completed with 3 local objects.
remote: error: Trace: 015ee68316c493da0756d3d6e06c403ad8d97b15f1496100acff905654bed8d2
remote: error: See https://gh.io/lfs for more information.
remote: error: File engine/grpcio-1.64.1-cp312-cp312-linux_aarch64.whl is 107.75 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
To github.com:grafana/oncall.git
 ! [remote rejected]   jorlando/update-uwsgi-version -> jorlando/update-uwsgi-version (pre-receive hook declined)
error: failed to push some refs to 'github.com:grafana/oncall.git'
```